rudder-web/src/main/webapp/templates-hidden/components/passwordInput.html

```diff
@@ -0,0 +1,120 @@
+              <select id="hashAlgo" class="form-control" ng-model="newPassword.hash" ng-options="prefix as hash for (prefix, hash) in hashes">
+              </select>
+               <span id="passwordHashHelp" class="help-block">
+                 This password will be hashed using the <strong>{{hashes[newPassword.hash]}}</strong> algorithm and stored and distributed only as a hash.
+                 The plain text value entered above will not be stored.
+               </span>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class=" panel bloc-action" ng-if="formType === 'preHashed' || formType === 'clearText'">
+        <div class="">
+          <div class="form-group has-feedback">
+            <label for="passwordHash{{formType === 'preHashed' ? '' : newPassword.show ? '-clear': '-hidden'}}" class=" control-label">New password</label>
+           <div class="input-group col-xs-12" ng-if=" formType === 'clearText'">
+
+              <input ng-if="newPassword.show" type="text" class="form-control toggle-type animate-show" id="passwordHash-clear" ng-model="newPassword.password" placeholder=""/>
+              <input ng-hide="newPassword.show" type="password" class="form-control toggle-type animate-show" id="passwordHash-hidden" ng-model="newPassword.password" placeholder=""/>
+              <span class="input-group-btn">
+                <button type="button" ng-click="newPassword.show = ! newPassword.show" class="btn btn-default reveal-password">
+                  <span ng-class="newPassword.show ? 'glyphicon-eye-close' : 'glyphicon-eye-open'" class="glyphicon "></span>
+                </button>
+              </span>
+            </div>
+           <div class="col-xs-12" ng-if=" formType === 'preHashed'">
+              <input type="text" class="form-control toggle-type" id="passwordHash" ng-model="newPassword.password" placeholder=""/>
+            </div>
+            <span class=" help-block col-xs-12">
+               This <strong class="variation">{{formType === 'preHashed' ? "hash" : "password"}}</strong> will be stored and distributed verbatim (plain text).
+             </span>
+          </div>
+        </div>
+      </div>
+      <div id="bloc-action-3" class=" panel bloc-action" ng-if="formType === 'script'">
+        <div class="">
+          <div class="form-group has-feedback">
+            <label for="script" class=" control-label">Script</label>
+           <div class="input-group col-xs-12 ">
+              <textarea class="form-control" ng-model="newPassword.password" rows="4" id="script"></textarea>
+              <span id="passwordHashHelp" class="help-block">
+                You can enter a JavaScript expression here in a <code>${eval ...}</code> block. See <a href="/rudder-doc">documentation</a> here for details.<br/>
+               Passwords will be computed for each node during policy generation. To use hashed passwords, make sure to call the hash() function in the script above.
```

Comment:
i'm not sure I understand the message
shoudln't it be rudder.password.something ?